code/game/objects/structures/divine.dm

```diff
@@ -1,3 +1,20 @@
+GLOBAL_LIST_INIT(potion_rates, list(
+	/obj/item/slimepotion/slime/sentience		=9,
```

Comment:
These I'd be very wary about. Sentience potions, afaik, can be used on any fo13 mob, meaning possible sentient deathclaws (which, thinking about it, I kinda like). Speed potions totally negate armor slowdown, making salvaged armor basically better than PA. Slimeradio... I think it just works on mobs, but it gives them a built in radio. And Im not sure if slime peace can be reversed.

I'd take out the slime speed, and maybe the peace potion depending on how it worms.

Reply:
Removed those potions

Reply:
thanks~